.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: node_js
 node_js:
-  - "0.10"
```

Comment:
Is there a reason to remove these? Some users might still be on Node v4. We can do this in a future PR instead.

Reply:
The version 4 is deprecated .